core/controllers/profile_test.py

```diff
@@ -43,6 +43,77 @@ def test_get_profile_page_of_existing_user(self):
         response = self.get_html_response('/profile/%s' % self.OWNER_USERNAME)
         self.assertIn(b'<oppia-root></oppia-root>', response.body)
 
+    def test_page_not_found(self):
+        def mock_false_function(*_):
+            """Mocks a function that returns False."""
+            return False
+        sm1 = 'Could not find the page '
+        sm2 = 'http://localhost/profilehandler/data/%s.' % self.EDITOR_USERNAME
+        message = sm1 + sm2
```

Comment:
You don't need to create three variables for this, just directly create the final one.

Reply:
I thought that might be an issue but I created them because there was a linting issue i.e 'Line too long'. But I'll find another way to do it.